packages/kata-sandbox-docker/Dockerfile

```
@@ -0,0 +1,55 @@
+  && command -v make \
+  && command -v python3
+
+RUN mkdir -p /usr/local/share/kata-sandbox-artifacts
+COPY artifacts/kata-code-cli.tgz /usr/local/share/kata-sandbox-artifacts/kata-code-cli.tgz
+COPY artifacts/codex.tgz /usr/local/share/kata-sandbox-artifacts/codex.tgz
+
+RUN printf '%s  %s\n' "${KATACODE_CLI_SHA256}" /usr/local/share/kata-sandbox-artifacts/kata-code-cli.tgz | sha256sum -c - \
+  && printf '%s  %s\n' "${CODEX_SHA256}" /usr/local/share/kata-sandbox-artifacts/codex.tgz | sha256sum -c -
+
+RUN node --input-type=module -e '
+  import { execFileSync } from "node:child_process";
```

Hadolint (2.15.1)

[error] unexpected 'i'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)